zipline/__main__.py

```diff
@@ -227,9 +235,18 @@ def run(ctx,
         broker,
         broker_uri,
         state_file,
+        broker_list,
         realtime_bar_target):
     """Run a backtest for the given algorithm.
     """
+
+    if broker_list:
+        click.echo("Supported brokers:")
+        for module_loader, name, ispkg in pkgutil.iter_modules(brokers.__path__):
```

Comment:
`module_loader` and `ispkg` variables seems to be unused.
You can avoid them with: `for _, name, _ in pkgutil.iter_modules(brokers.__path__)`

Reply:
Just noticed that this line is too long. Zipline runs flake8 with 79 line width allowed:
https://travis-ci.org/zipline-live/zipline/jobs/285172438#L1206